src/reduxStore/addressSlice.js

```diff
@@ -82,13 +77,22 @@ export const updateLastZone = (value) => ({
   },
 });
 
-const addressReducer = (state = initialState, action = null) => {
+// export const updateMarker = (value) => {
+//     return {
+//         type: 'UPDATE_MARKER',
+//         payload: {
+//             value,
+//         },
+//     };
+// };
+
+const addressReducer = (state = initialState, action) => {
```

Comment:
the default parameters should be last (action, state = initialState)

Reply:
good catch, i just pushed a fix